packages/react/src/components/numericInput/NumericInputActions.ts

```diff
@@ -25,12 +24,12 @@ const unmount = (id: string): IReduxAction<{id: string}> => ({
 
 export interface ISetNumericInputPayload {
     id: string;
-    value: ReactText;
+    value: number;
```

Comment:
Because of this I would mark the commit as breaking

Reply:
done